Documentation/BuildInstructions.md

```diff
@@ -3,33 +3,41 @@
 ### Prerequisites
 
 #### Linux prerequisites
-Make sure you have all the dependencies installed:
+Make sure you have all the dependencies installed (`ninja` is optional, but is faster in practice):
 
 **Debian / Ubuntu**
 ```bash
-sudo apt install build-essential cmake curl libmpfr-dev libmpc-dev libgmp-dev e2fsprogs qemu-system-i386 qemu-utils
+sudo apt install build-essential cmake curl libmpfr-dev libmpc-dev libgmp-dev e2fsprogs ninja qemu-system-i386 qemu-utils
```

Comment:
This is wrong! See:

https://github.com/SerenityOS/serenity/blob/13d7c09125f8eec703d0a43a9a87fc8aa08f7319/.github/workflows/cmake.yml#L155

and

https://packages.ubuntu.com/search?keywords=ninja